tests/phpunit/tests/general/wpGetDocumentTitle.php

```diff
@@ -60,7 +60,7 @@ public function add_title_tag_support() {
 	public function test__wp_render_title_tag() {
 		$this->go_to( '/' );
 
-		$this->expectOutputString( sprintf( "<title>%s &#8211; %s</title>\n", $this->blog_name, get_option( 'blogdescription' ) ) );
+		$this->expectOutputString( sprintf( "<title>%s</title>\n", $this->blog_name ) );
```

Comment:
It'd be good to have versions with and without the blog description if there aren't already.

Reply:
Added in [4cc6a34](https://github.com/WordPress/wordpress-develop/pull/3050/commits/4cc6a346e100faa012ed68f533c9d7610e71a586) in [PR 3050](https://github.com/WordPress/wordpress-develop/pull/3050).